src/App.css

```diff
@@ -121,3 +121,10 @@ background-size: cover;
   margin-top: 20px;
 }
 
+.MuiInputBase-root {
+  color: rgba(255, 255, 255, 1) !important;
```

Comment:
Its a good practice to move the base app colors to a separate file somewhere in shared folder and reference them.
Example:

File name : Colors.ts

const BaseColors = {
 Black: '#000',
}

then import that color inside needed files and use it.